CMakeLists.txt

```diff
@@ -273,7 +276,7 @@ install(TARGETS ttnn
 )
 if(WITH_PYTHON_BINDINGS)
     # Install .so into src files for pybinds implementation
-    install(FILES ${PROJECT_BINARY_DIR}/lib/_ttnn.so
```

Comment:
How does the install target know which file to copy over? I don't see this declared anywhere else

Reply:
The CMake target "ttnn" is a shared library target. CMake will always know where it is building it, so we don't have to tell CMake where it is. We just need to say "install target", just like we are doing a few lines above (line 272).